app/controllers/cip_change_alert_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CipChangeAlertController < ApplicationController
+  before_action :authenticate_user!
+
+  def start
+  end
+
+  def acknowledge_start
+    current_user.cip_change_message&.destroy!
```

Comment:
Do we need `current_user` here? `cip_change_message` is a helper method already :)